<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Plot_Definer_2D</class>
 <widget class="QWidget" name="Plot_Definer_2D">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="0">
    <widget class="QLabel" name="label">
     <property name="toolTip">
      <string>Set the exact x coordinate of the top left corner of the plot window in pixels (min 0)</string>
     </property>
     <property name="text">
      <string>Top Left X</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_9">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>x-label:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="3">
    <widget class="QLineEdit" name="lineEdit_plot_height">
     <property name="toolTip">
      <string>Minimum is 126 pixels</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="label_10">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>z-label:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>y-axis:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLineEdit" name="lineEdit_zlabel"/>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="lineEdit_top_left_x">
     <property name="toolTip">
      <string>Minimum is 0 pixels</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QLabel" name="label_3">
     <property name="toolTip">
      <string>Set the width of the plot window in pixels (min 430)</string>
     </property>
     <property name="text">
      <string>Plot Width</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>y-label:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QLabel" name="label_4">
     <property name="toolTip">
      <string>Set the height of the plot window in pixels (min 126)</string>
     </property>
     <property name="text">
      <string>Plot Height</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_11">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>title:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_7">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>z-axis:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="Variable_Box" name="lineEdit_x_axis"/>
   </item>
   <item row="8" column="3">
    <widget class="QLineEdit" name="lineEdit_plot_width">
     <property name="toolTip">
      <string>Minimum is 430 pixels</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>x-axis:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_2">
     <property name="toolTip">
      <string>Set the exact y coordinate of the top left corner of the plot window in pixels (min 0)</string>
     </property>
     <property name="text">
      <string>Top Left Y</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QLineEdit" name="lineEdit_n_data_points">
     <property name="text">
      <string>inf</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="Variable_Box" name="lineEdit_y_axis"/>
   </item>
   <item row="4" column="1">
    <widget class="Variable_Box" name="lineEdit_z_axis"/>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEdit_title"/>
   </item>
   <item row="2" column="3">
    <widget class="QLineEdit" name="lineEdit_xlabel"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_12">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string># data points:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLineEdit" name="lineEdit_top_left_y">
     <property name="toolTip">
      <string>Minimum is 0 pixels</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLineEdit" name="lineEdit_ylabel"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_13">
     <property name="styleSheet">
      <string notr="true">QLabel {
    rgb(134, 134, 134);
}</string>
     </property>
     <property name="text">
      <string>Plot Position</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Variable_Box</class>
   <extends>QLineEdit</extends>
   <header>nomad_camels.ui_widgets.variable_tool_tip_box</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit_title</tabstop>
  <tabstop>lineEdit_x_axis</tabstop>
  <tabstop>lineEdit_y_axis</tabstop>
  <tabstop>lineEdit_z_axis</tabstop>
  <tabstop>lineEdit_xlabel</tabstop>
  <tabstop>lineEdit_ylabel</tabstop>
  <tabstop>lineEdit_zlabel</tabstop>
  <tabstop>lineEdit_n_data_points</tabstop>
  <tabstop>lineEdit_top_left_x</tabstop>
  <tabstop>lineEdit_top_left_y</tabstop>
  <tabstop>lineEdit_plot_width</tabstop>
  <tabstop>lineEdit_plot_height</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
